pkg/propertyprovider/aks/provider.go

```diff
@@ -41,13 +40,9 @@ const (
 	PerGBMemoryCostProperty = "kubernetes.azure.com/per-gb-memory-cost"
 
 	// The resource properties.
-	TotalCPUCapacityProperty       = "resources.kubernetes-fleet.io/total-cpu"
-	AllocatableCPUCapacityProperty = "resources.kubernetes-fleet.io/allocatable-cpu"
-	AvailableCPUCapacityProperty   = "resources.kubernetes-fleet.io/available-cpu"
-
-	TotalMemoryCapacityProperty       = "resources.kubernetes-fleet.io/total-memory"
-	AllocatableMemoryCapacityProperty = "resources.kubernetes-fleet.io/allocatable-memory"
-	AvailableMemoryCapacityProperty   = "resources.kubernetes-fleet.io/available-memory"
+	// Available CPU and memory resource properties.
+	AvailableCPUCapacityProperty    = "resources.kubernetes-fleet.io/available-cpu"
```

Comment:
I wonder why is this not moved to the common too?

Reply:
Moved as discussed previously.